package.json

```diff
@@ -147,7 +147,7 @@
     "react-svg-loader": "^2.1.0",
     "react-test-renderer": "^15.4.1",
     "rimraf": "^2.5.4",
-    "sass-loader": "^6.0.6",
+    "sass-loader": "^7.0.3",
```

Comment:
Not so sure about this one. I've had intermittent failures when we tried this before